### PR TITLE
[types/google-apps-script] Adding Types manually

### PR DESCRIPTION
**Pull request by oshliaer**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/apps-script/guides/web#request_parameters

Please, consider the suggestion to add `doGet` & `doPost` types as a part of the Google Apps Script Events model.

### CODE REVIEW

File: types/google-apps-script/google-apps-script-tests.ts [hunk 1]
```diff
@@ -28,65 +30,76 @@ CalendarApp.GuestStatus.NO;
...
 // test for URLFetchRequestOptions.payload
 import URLFetchRequestOptions = GoogleAppsScript.URL_Fetch.URLFetchRequestOptions;
-const postTest = (payload: object): string => {
-  const url = 'http://httpbin.org/post';
-  const params: URLFetchRequestOptions = {
-    method: 'post',
-    payload: payload
-  };
-  return UrlFetchApp.fetch(url, params).getContentText();
+const postTest = (payload: Object): string => {
```
Comment (grant) [inline types/google-apps-script/google-apps-script-tests.ts]: Lowercase please.

File: types/google-apps-script/google-apps-script-events.d.ts [hunk 2]
```diff
@@ -20,6 +21,29 @@ declare namespace GoogleAppsScript {
...
+     * isGoogleType, requireContentType, requireData, requireName, setBytes,
+     * setContentType, setContentTypeFromExtension, setDataFromString,
+     * setDataFromString, setName, toString
+     */
+    interface AppsScriptHttpRequestEventPostData {
+        length: number,
+        type: string,
+        contents: string,
+        name: string, // Always the value "postData"
+      }
```
Comment (grant) [inline types/google-apps-script/google-apps-script-events.d.ts]: Bad formatting

File: types/google-apps-script/google-apps-script-events.d.ts [hunk 3]
```diff
@@ -20,6 +21,29 @@ declare namespace GoogleAppsScript {
...
+    interface AppsScriptHttpRequestEvent {
+        parameter: object,
+        contextPath: string,
+        contentLength: number
+        queryString: string,
+        parameters: object,
+      }
+
+    /**
+     * TODO: What about other properties? Object.keys(e.postData) contains:
```
Comment (grant) [inline types/google-apps-script/google-apps-script-events.d.ts]: No todos in PRs

**Review by grant — CHANGES_REQUESTED** [review]
Thanks. Can you brush up this PR? Looks good in general.

File: types/google-apps-script/google-apps-script-tests.ts (hunk 1 above)
Comment (oshliaer) [reply types/google-apps-script/google-apps-script-tests.ts]: You're right! Thanks!

File: types/google-apps-script/google-apps-script-events.d.ts (hunk 2 above)
Comment (oshliaer) [reply types/google-apps-script/google-apps-script-events.d.ts]: I returned the formatting as it is in this directory.

File: types/google-apps-script/google-apps-script-events.d.ts (hunk 3 above)
Comment (oshliaer) [reply types/google-apps-script/google-apps-script-events.d.ts]: Sorry. I'll be more attentive.

**Review by grant — CHANGES_REQUESTED** [review]
Can you please undo the formatting changes?
Use 2 spaces.

**Review by grant — APPROVED** [review]

**Comment by oshliaer** [comment]
@grant 

Should I undo all formatting?

Looks like I added something extra.

**Comment by grant** [comment]
Looks fine.

In general, have the smallest diff possible when adding features.

**Review by grant — APPROVED** [review]

**Pull request merged**